packages/framework/presence/README.md

```diff
@@ -47,7 +47,7 @@ Notifications value managers are special case where no data is retained during a
 While this package is developing as experimental and other Fluid Framework internals are being updated to accommodate it, a temporary Shared Object must be added within container to gain access.
 
 ```typescript
-import { acquirePresenceViaDataObject, ExperimentalPresenceManager } from "@fluid-experimental/presence";
+import { acquirePresenceViaDataObject, ExperimentalPresenceManager } from "@fluidframework/presence";
```

Comment:
I would have expected the auto-generated contents lower in this file to have changed as a result of the package rename / "exports" changes. You may need to run `npm run build:readme` from the root of the repo to regenerate contents here.

I _thought_ the PR pipeline was configured to catch this, but maybe not?
@tylerbutler FYI

Reply:
After some digging, this appears to be an issue in our markdown-magic library. This README is using an old template that we removed, but `markdown-magic` is not failing on it. I'll investigate this.